tests/openedx_tagging/core/tagging/test_api.py

```diff
@@ -913,3 +913,183 @@ def test_get_object_tag_counts_deleted_disabled(self) -> None:
         assert tagging_api.get_object_tag_counts("object_*") == {obj1: 1, obj2: 2}
         tagging_api.add_tag_to_taxonomy(self.taxonomy, "DPANN", parent_tag_value="Archaea")
         assert tagging_api.get_object_tag_counts("object_*") == {obj1: 2, obj2: 2}
+
+    def test_copy_tags(self) -> None:
+        obj1 = "object_id1"
+        obj2 = "object_id2"
+
+        tags_list = [
+            {
+                "value": "English",
+                "taxonomy": self.language_taxonomy,
+            },
+            {
+                "value": "DPANN",
+                "taxonomy": self.taxonomy,
+            },
+        ]
+
+        for tag_object in tags_list:
+            tagging_api.tag_object(object_id=obj1, taxonomy=tag_object["taxonomy"], tags=[tag_object["value"]])
+
+        tagging_api.copy_tags(obj1, obj2)
+
+        object_tags = tagging_api.get_object_tags(obj2)
+
+        assert len(object_tags) == 2
+        for index, object_tag in enumerate(object_tags):
+            object_tag.full_clean()
```

Comment:
```suggestion

```
Is `full_clean` required here?

Reply:
Yes, we use `full_clean()` to validate the object tag in the test. It's used in other tests: [[1](https://github.com/openedx/openedx-learning/blob/main/tests/openedx_tagging/core/tagging/test_api.py#L430)][[2](https://github.com/openedx/openedx-learning/blob/main/tests/openedx_tagging/core/tagging/test_api.py#L449)][[3](https://github.com/openedx/openedx-learning/blob/main/tests/openedx_tagging/core/tagging/test_api.py#L479)]